kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/StampViewModel.java

```diff
@@ -101,4 +102,15 @@ public List<ConceptEntity<ConceptEntityVersion>> findAllPaths(ViewProperties vie
             return List.of();
         }
     }
+
+    /**
+     *  We are loading only active and Inactive states in the stamp editor.
+     * @return list of statuses
+     */
+    private List<ConceptEntity> loadOnlyActiveInactiveStatues() {
+        List<ConceptEntity> statuses = new ArrayList<>();
+        statuses.add((ConceptEntity) Entity.getFast(ACTIVE_STATE));
+        statuses.add((ConceptEntity) Entity.getFast(INACTIVE_STATE));
+        return statuses;
```

Comment:
Since these options are in bindings why do they have to be queried from the entity service?